swxsoc/__init__.py

```diff
@@ -0,0 +1,34 @@
+# see license/LICENSE.rst
+try:
+    from ._version import version as __version__
+    from ._version import version_tuple
+except ImportError:
+    __version__ = "unknown version"
+    version_tuple = (0, 0, "unknown version")
+
+from swxsoc.util.config import load_config, print_config
+from swxsoc.util.logger import _init_log
+
+# Load user configuration
+config = load_config()
+
+log = _init_log(config=config)
+
+# Then you can be explicit to control what ends up in the namespace,
+__all__ = ["config", "print_config"]
+
+MISSION_NAME = "swxsoc"
```

Comment:
We'll need to update these, but there is code in the `util.create_science_filename` and `swxdata.SWXData.global_attribute_template` functions that does validation against these. We might need to move these to a configuration file rather than having them as package constants so they can be more easily overwritten by extending packages and classes.